tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -135,6 +135,24 @@
                 </methods>
             </class>
             <class name="org.ballerinalang.test.expressions.binaryoperations.EqualAndNotEqualOperationsTest"/>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalNaNInfinityTest">
+                <!--We can remove excludes once 'isNaN' and 'isFinite' are supported-->
+                <methods>
+                    <exclude name="testIsFinite"/>
+                    <exclude name="testIsNaN"/>
+                </methods>
+            </class>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalValueNegativeTest"/>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalBFloatComparisonTest"/>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalTypeConversionTest"/>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalUsageTest"/>
+            <class name="org.ballerinalang.test.types.decimaltype.BDecimalValueTest">
+                <!--Unary + and - is not supported yet-->
```

Comment:
These two ops should be straight forward to add. Could you also add them so that we can complete the decimal type end-to-end?